conans/client/conan_api.py

```diff
@@ -845,7 +846,8 @@ def remote_rename(self, remote_name, new_new_remote):
 
     @api_method
     def remote_list_ref(self):
-        return self._registry.refs.list
+        return {ConanFileReference.loads(r): remote_name
```

Comment:
This is a ConanFileReference going outside of the API, instead of a str. Can't recall if this was intended design.

Reply:
Neither do I. Changing to str just in case